app/assets/javascripts/components/Filters.js

```diff
@@ -43,5 +55,53 @@ define(['jquery', 'DoughBaseComponent'], function($, DoughBaseComponent) {
     this.$form.submit();
   }
 
+  /**
+  * Assign/Remove listner on toggle
+  */
+  Filter.prototype._toggleFilters = function(self, isLoad) {  
+    if ( $(window).width() < this.breakpoint ) {
+      self._toggleMobile(self, isLoad);
+    } else {
+      self._toggleDesktop(self, isLoad);
+    }
+    self.windowSize = $(window).width();
+  } 
+
+  /**
+  * Is Mobile
+  *   was previously desktop, or page just loaded, so rest for mobile
+  */
+  Filter.prototype._toggleMobile = function(self, isLoad) {
+    if ( self.windowSize >this.breakpoint || isLoad ) {
```

Comment:
I would format the line with less spaces around the if and spaces for the comparison like.`if (self.windowSize > this.breakpoint || isLoad)`. But that's just my taste.

Reply:
done